### ISSUE TITLE
config.yaml file usage

### ISSUE THREAD

**Issue by gari-marcos** [xingdi-eric-yuan/GATA-public#8]
Hello, I am trying to start up the code but I am having some problems with the ```config.yaml``` file, as I don't know how to configure the parameters depending on the phase I am running. For instance, regarding the ```checkpoint``` parameters, as I saw the agent checks if the ```load_pretrained``` parameter is set to ```True``` and depending on that it loads the ```exp_tag.json```. However, after running every train script I don't get any ```pretrained_model.pt```. Could you please show me a step-by-step to use the ```config```file properly? 
Thank you so much.

**Comment by MarcCote** [comment]
@xingdi-eric-yuan we should take advantage of the command line `--params` argument and provide the actual settings we used in the README (instead of saying `# Modify config.yaml`).

**Comment by xingdi-eric-yuan** [comment]
If `load_pretrained` is set to `True`, the agent will try to find a file with name defined in `load_from_tag` to load. As Marc said, we will provide a more detailed settings but on high level one should:

1. During self-supervised pre-training: set `experiment_tag` to be, say, `my_dgi_pretrain`, set `load_pretrained` to be `False`. The system will then ignore the `load_*` configs. After every a period of time (`report_frequency`), a checkpoint file `my_dgi_pretrain_model.pt` will be saved; also the latest results are updated into `my_dgi_pretrain.json`
2. During RL training: set `experiment_tag` to be , say `my_rl_training`, set `load_pretrained` to be `True`, and `load_from_tag` to be `my_dgi_pretrain_model`. Then the system will load the pre-trained parameters that contain `load_parameter_keywords` in their names. Again, after every a period of time (`report_frequency`), a checkpoint file `my_rl_training_model.pt` will be saved; also the latest results are updated into `my_rl_training.json`

**Comment by gari-marcos** [comment]
Thank you so much! I am running it now, but I'm not pretty sure about the ```evaluate``` field. When I enable it at the pretraining it gets stuck and doesn't go on. Do I have to enable it only when running ```test_rl.py```?
Thank you again.

**Comment by MarcCote** [comment]
@gari-marcos thank you very much for helping us making this code more reproducible.
